softwarereview_policies.es.Rmd

```diff
@@ -3,276 +3,271 @@
+
+-   La comunicación entre quienes enviaron el paquete, quienes lo revisan y editan tendrá lugar principalmente en GitHub, aunque puedes optar por ponerte en contacto con quien hace la edición por correo electrónico o por Slack para algunas cuestiones.
+    Cuando envíes un paquete, asegúrate de que tu configuración de notificaciones de GitHub sea correcta para que no te pierdas un comentario.
+
+-   Quien hace el envío del paquete puede elegir que su envío se ponga en espera (se le aplica la etiqueta *holding*).
+    El estado de espera se revisará cada 3 meses, y después de un año el *issue* se cerrará.
+
+-   Si quien sea responsable del paquete no ha solicitado una etiqueta de espera, sino que simplemente no responde, podremos cerrar el tema en el plazo de un mes desde el último intento de contacto.
+    Este intento incluirá un comentario etiquetando a la persona, pero también un correo electrónico utilizando la dirección de correo electrónico que aparece en el archivo *DESCRIPTION* del paquete.
+    Este es uno de los raros casos en los que quien hace la edición intentará ponerse en contacto con el autor por correo electrónico.
+
+-   Para volver a enviar un paquete cuyo envío fue cerrado hay que iniciar un nuevo envío.
+    Si el paquete sigue siendo relevante, la persona tendrá que responder a las revisiones iniciales antes de que la persona encargada de la revisión empiece a buscar nuevas personas para la revisión.
 
 ### Publicar en otros lugares
 
-- Te recomendamos encarecidamente que envíes tu paquete para su revisión *antes de* publicar
-  en CRAN o enviar un artículo de software que describa el paquete a una revista.
-  Las devoluciones de las revisiones pueden dar lugar a importantes mejoras y actualizaciones de tu paquete,
-  incluyendo el cambio de nombre y los cambios en las funciones.  No consideramos
-  que la publicación previa en CRAN o en otros lugares sea razón suficiente para
-  no adoptar las recomendaciones de los revisores o editores.
-- No envíes tu paquete para su revisión mientras éste o un manuscrito asociado
-  también está siendo revisado en otro lugar, ya que esto puede dar lugar a solicitudes de cambios
-  solicitudes de cambios.
+-   Te recomendamos fuertemente que envíes tu paquete para su revisión *antes de* publicarlo en CRAN o enviar un artículo de software que describa el paquete a una revista.
+    Las devoluciones de las revisiones pueden dar lugar a importantes mejoras y actualizaciones de tu paquete, incluyendo el cambio de nombre y funcionalidad en las funciones.
+    No consideramos que la publicación previa en CRAN o en otros lugares sea razón suficiente para no adoptar las recomendaciones de quienes hacen la revisión o edición.
 
-### Conflicto de intereses para revisores/editores {#coi}
+-   No envíes tu paquete para su revisión mientras éste o un manuscrito asociado también esté siendo revisado en otro lugar, ya que esto puede dar lugar a pedidos de cambios conflictivas.
```

Comment:
Me parece que "manuscrito" se está usando para traducir paper.  Puede ser?  No sería mas adecuado usar artículo científico en vez de manuscrito?

Reply:
@eliocamp ?

Reply:
Podemos usar artículo científico, no recuerdo que hicimos previamente. Es para agregar al glosario y chequear en la revisión global!

Reply:
No recuerdo esta situación en los que ya he mirado, pero en esta sección aparece muchas veces, capaz que por eso lo noté.  Yo hago los cambios en este, si vos podes agregarlo al glosario.